tests/integration/plugin_test.go

```diff
@@ -0,0 +1,24 @@
+//+build integration
+//+build !windows
+
+package integration
```

Comment:
This whole files spacing is messed up; it should be consistent with the common spacing of this project.

Reply:
fixed